interfaces/builtin/serial_port.go

```diff
@@ -43,7 +43,7 @@ func (iface *SerialPortInterface) String() string {
 
 // Pattern to match allowed serial device nodes, path attributes will be
 // compared to this for validity when not using udev identification
-var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty[A-Z]{1,3}[0-9]{1,3}$")
+var serialDeviceNodePattern = regexp.MustCompile("^/dev/tty[a-zA-Z0-9]+$")
```

Comment:
Should it filter out the case without trailing device number? eg. /dev/ttyUSB
The regexp can be "^/dev/tty[a-zA-Z0-9]+[0-9]+$"